docs/guides/blocks/sync-tables/index.md

```diff
@@ -128,7 +128,7 @@ In some areas of the UI, such as the settings pane, the name will be rewritten t
     Changing the name of a sync table will break any existing docs that use it. When creating your Pack select your names carefully.
 
 
-## Identity
+## Identity {: #identity}
 
 Every sync table is required to specify an [`identityName`][identityName], which is a unique identifier for the type of item being synced in that table. By convention this is usually the singular form of the name of the item being synced. For instance, a "Tasks" table would use an identity name of "Task". This identity name is used to reference the table when [creating row references](#references) or [updating rows with actions](#actions).
```

Comment:
I've tried to keep the regular sync tables guide free of dynamic sync table details, to avoid overloading the reader. I'll take an AI to add this detail to the dynamic sync tables guide, as I have another update I want to do there outside this migration work.